lib/MetaCPAN/Web/Model/ReleaseInfo.pm

```diff
@@ -283,12 +283,22 @@ sub normalize_issue_url {
 sub normalize_notification_type {
     my ( $self, $data ) = @_;
     if ( is_hashref($data) ) {
-        if (   is_hashref( $data->{permission} )
-            && is_arrayref( $data->{permission}{co_maintainers} ) )
-        {
-            for ( reverse @{ $data->{permission}{co_maintainers} } ) {
-                if ( $_ =~ m/^(NEEDHELP|ADOPTME|HANDOFF)$/ ) {
-                    return $_;
+        if ( is_hashref( $data->{permission} ) ) {
+            my %special = (
+                NEEDHELP => 1,
+                ADOPTME  => 1,
+                HANDOFF  => 1
+            );
+            if (   $data->{permission}->{owner}
+                && $special{ $data->{permission}->{owner} } )
```

Comment:
I think `exists $special{ $data->{permission}->{owner} }` would be the safest way to do this.

Reply:
sure I've added a defined and an exists.